### ISSUE TITLE
Change password issue

### ISSUE THREAD

**Issue by aliakakis** [StateOfJS/StateOfJS-Vulcan#4]
When I select to change my password I get a field named "Email:" with my password written beside it.

**Comment by SachaG** [comment]
Can you post a screenshot? I think that might be a problem with your browser's autofill, as we don't store passwords in plain text and don't have access to them.

**Comment by aliakakis** [comment]
Yes of course. Apologies for not posting a screen earlier, I had to restart first for an update :). I have erased parts of my password. I am using Firefox latest version. Steps I am doing to reproduce it are Login -> Account -> Change password (all seems OK) -> Cancel change -> Click to change again -> Below screen appears.

![image](https://user-images.githubusercontent.com/2565074/69611551-afaf4880-1036-11ea-94c0-bd9728528d38.png)

**Comment by aliakakis** [comment]
@SachaG I am also getting the following screen after sign out

![image](https://user-images.githubusercontent.com/2565074/69612074-b7232180-1037-11ea-9625-45fd861eb961.png)

**Comment by aliakakis** [comment]
I am using LastPass. It seems as if it writes my password in the Email field. I have changed my password but seems to continue showing my previous one.

**Comment by SachaG** [comment]
Oh ok, when the email is prefilled through the URL (`?email=foo@bar.com`) it shows up as `Email: foo@bar.com`. I guess LastPass is messing with that feature somehow. What happens if you click "change", can you enter the proper email?

**Comment by aliakakis** [comment]
When I click change it shows the Enter password input field not the email field.

**Comment by aliakakis** [comment]
I also get the following error message in the console. Please note that I am behind a proxy.

![image](https://user-images.githubusercontent.com/2565074/69614172-5b5a9780-103b-11ea-9c60-588dc792a3d1.png)

**Comment by jtrollia** [comment]
Same issue here, I'm not using LastPass.
